main.py

```diff
@@ -158,6 +174,11 @@ def main():
     respond_thread.daemon = True
     respond_thread.start()
 
+    # audio_response_thread = threading.Thread(target=global_vars.audio_player.play_audio_loop,
```

Comment:
commented code?

Reply:
It is required for the next PR which contains an enhancement request. Easier to comment it here and remove the comment when that PR will be put up.